src/TLuaInterpreter.cpp

```diff
@@ -15639,23 +15645,33 @@ int TLuaInterpreter::addMouseEvent(lua_State * L)
         actionInfo << lua_tostring(L, 3);
     }
 
-    Host& host = getHostFromLua(L);
+    // tooltip text
+    if (!lua_isstring(L, 4)) {
+        actionInfo << "";
```

Comment:
:information_source: I think you should be using `QString()` instead of `""` see **[QString: distinction between null and empty strings](https://doc.qt.io/qt-5/qstring.html#distinction-between-null-and-empty-strings)**...